src/components/comment/demo/data.ts

```diff
@@ -1,7 +1,5 @@
-const Jabber = require('jabber');
-const random = require('lodash/random');
-const sample = require('lodash/sample');
-const uniqueId = require('lodash/uniqueId');
+import Jabber from 'jabber';
```

Comment:
Hi @tylersticka , thanks for using Jabber, I have added ts support in v.1.5.2.
regards 
Anil